source/_patterns/02-molecules/card/_card.twig

```diff
@@ -43,66 +39,62 @@
   {% endif %}
 
   {% if card_image_location == 'top' %}
-
     {# if `card_image` is null, this falls back to a holderjs placeholder #}
     {% include '@atoms/image/_image.twig' with {
       imageStyleName: 'card',
       class: 'card-img-top',
       img: card_image,
     } %}
-
   {% endif %}
 
   {% if card_image_overlay %}
     <div class="card-img-overlay">
       <p>{{ card_image_overlay }}</p>
     </div>
   {% endif %}
-  <div class="card-body">
-    <h4 class="card-title">{{ card_title }}</h4>
-    {% if card_subtitle %}
-      <h6 class="card-subtitle mb-2 text-muted">{{ card_subtitle }}</h6>
-    {% endif %}
-    {% if card_text %}
-      <p class="card-text {{ text_color }}">{{ card_text }}</p>
-    {% endif %}
-    {% if card_link %}
-      <a href="{{ card_link }}" class="card-link">{{ card_link_text }}</a>
-    {% endif %}
-    {% if button_text and button_color %}
-      <div>
-        {% include "@atoms/button/_button.twig" with {
-          button_element: button_element,
-          button_text: button_text,
-          button_color: button_color,
-          button_link: button_link,
-        } only %}
-      </div>
-    {% endif %}
+
+  {% if card_flush %}
+    {# Note that 'p2card--flush--body' is not a bootstrap class. It only being used to designate the flush body#}
+    {% set body_class = 'p2card--flush--body' %}
```

Comment:
This isn't a BEM name. Does this class just not apply padding?

Reply:
Would it be more applicable to make this a utility class that could be used either: a) only within PL; or b) globally by the system (possibly tied to something BS provides). I'm still not sure there's a need for a BEM style class at all here if it is only for demo purposes (if I'm understanding why it is being used).